src/meta/processors/jobMan/JobManager.h

```diff
@@ -113,13 +134,13 @@ class JobManager : public nebula::cpp::NonCopyable, public nebula::cpp::NonMovab
     // The job in running or queue
     folly::ConcurrentHashMap<JobID, JobDescription>    inFlightJobs_;
 
-    std::unique_ptr<thread::GenericWorker>             bgThread_;
+    std::thread                                        bgThread_;
```

Comment:
What is the issue of `GenericWorker`?

Reply:
got some weird dead lock. And I believe I just need a simple thread to run "scheduleThread".

Reply:
👌